src/gen-lua-core-h.awk

```diff
@@ -23,6 +23,7 @@ BEGIN {
 
 }
 
+/^typedef struct CallInfo CallInfo;/ { next }  # Skip repeated typedefs
```

Comment:
I agree this is much better than the other PR.

The only nitpick is that we already have logic for skipping lines with an include. But here we're using "next" and there we're using "if not then print". I don't have a preference but it would be nice if they worked the same way.